src/pages/order-item/order-item.js

```diff
@@ -47,65 +52,101 @@ const OrderItem = ({ id }) => {
         false
       );
     } else {
-      dispatch(updateOrder(order))
+      dispatch(updateOrder(order));
     }
-  }
+  };
 
-  const {handleChange,values, handleSubmit, setFieldValue, dirty, initialValues, resetForm} = useFormik({
+  const {
+    handleChange,
+    values,
+    handleSubmit,
+    setFieldValue,
+    dirty,
+    initialValues,
+    resetForm
+  } = useFormik({
     initialValues: {},
-    onSubmit: handleFormSubmit,
-  })
+    onSubmit: handleFormSubmit
+  });
 
   useEffect(() => {
-    if(id) {
-      dispatch(getOrder(id))
+    if (id) {
+      dispatch(getOrder(id));
     }
-  }, [dispatch,id]);
+  }, [dispatch, id]);
 
-  useEffect(()=> {
-    if(selectedOrder) {
+  useEffect(() => {
+    if (selectedOrder) {
       resetForm({ values: selectedOrder });
     }
-  },[selectedOrder,resetForm])
+  }, [selectedOrder, resetForm]);
 
-  const formikHandleChange = ['CREATED','CONFIRMED'].includes(selectedOrder && selectedOrder.status) ? handleChange : ()=> {}
+  const formikHandleChange = ['CREATED', 'CONFIRMED'].includes(
```

Comment:
You use [...] array in 2 different places. Maybe make sense move it to separate variable

Reply:
done